components/gallery/GalleryItemButton/GalleryItemMoreActionBtn.vue

```diff
@@ -7,23 +7,52 @@
         v-if="mimeType.includes('image') && ipfsImage"
         item="Download"
         @click.native="downloadMedia" />
+      <NeoDropdownItem
+        v-if="currentOwner"
```

Comment:
How about displaying the burn button only when you are the onwer of the nft ?

Reply:
meh.. so no disabled btn then

Reply:
yes, according to the comment https://github.com/kodadot/nft-gallery/issues/4763#issuecomment-1396943625 , we do need the disabled button.

Reply:
### current implementation
- show burn btn when there is currentOwner (small hack for Nuxt parent/child lifecycle)
- disable burn btn when `currentOwner !== accountId`

What would you like me to do here?

Reply:
I would like to show burn btn when `currentOwner === accountId`, if I understand correct from the comment below.
<img width="941" alt="image" src="https://user-images.githubusercontent.com/31397967/214062798-081b2249-139e-473c-b50e-817c6bcff69b.png">

Reply:
On figma, [Gallery item](https://www.figma.com/file/gHSzqGPeQ9qK4L0qIDcTjY/gallery-item-handoff)
![Capture d’écran 2023-01-23 à 3 37 01 PM](https://user-images.githubusercontent.com/9987732/214066470-f12ff621-6c7f-4456-8cd2-8e79d1cc1b3c.png)

Reply:
![601](https://user-images.githubusercontent.com/9987732/214066965-45b5c125-3921-452d-9abe-9c404a4fb880.jpg)

Reply:
updated

Reply:
I just feel strange when I'm not the owner but could see the operation button that is only for the owner.
Whatever, if you feel normal let's move on!
<img width="829" alt="image" src="https://user-images.githubusercontent.com/31397967/214068729-44f686db-5a45-4d0f-afb4-ca972db21047.png">

Reply:
please don't mind my precedent comments, code respect figma design now

Reply:
Uhmm, you should not see the burn option when you are not the owner. The burn option is only visible when you are owner + listed/unlisted. Disabled state is not enough in this case. 
@Jarsen136 the whole issue was about owner ui, there was an issue with the hover effects and button type. 
Sorry if it was not clear.